src/python/fuzzing/testcase_manager.py

```diff
@@ -437,6 +441,10 @@ def run_testcase_and_return_result_in_queue(crash_queue,
     # Run testcase and check whether a crash occurred or not.
     return_code, crash_time, output = run_testcase(thread_index, file_path,
                                                    gestures, env_copy)
+    # TODO(flowerhack): Update this once Fuchsia knows how to properly extract
+    # crash time?
+    if crash_time is None:
```

Comment:
is this necessary? CF should automatically get the crash time based on observing when the process crashes.